### PR TITLE
remove register command

### PR DESCRIPTION
**Pull request by annyhe**

### CODE REVIEW

**Comment by annyhe** [comment]
@iamigo The readme changes have been merged. Please review.

**Comment by harshkothari410** [comment]
removing registry.json file is part of this story ? @annyhe @iamigo

File: src/config/config.js [hunk 1]
```diff
@@ -47,8 +47,6 @@ function getDefaultConfig() {
 
   const metadata = common.getCurrentMetadata();
   Object.assign(conf.collectorConfig, metadata);
-
-  debug('Initialized config: %s', JSON.stringify(conf));
```
Comment (annyhe) [inline src/config/config.js]: this is a near-duplicate of line 61 in src/config/config.js

**Comment by jgraff2** [comment]
@harshkothari410 @annyhe 
That's correct, it's no longer needed, but the registry.json isn't in git because it's collector-specific, right? So I don't think there's anything to remove.

**Pull request merged**